sdk/modelsrepository/azure-iot-modelsrepository/README.md

```diff
@@ -1,186 +1,3 @@
-# Azure IoT Models Repository client library for Python
+# Microsoft Azure SDK for Python
```

Comment:
I don't think this is the entire Microsoft Azure SDK for Python. This is one library.

Reply:
OOB: We're following our new deprecation process here - so it's a standard rename across all deprecated packages. [azure-sdk-for-python/doc/deprecation_process.md at main · Azure/azure-sdk-for-python (github.com)](https://github.com/Azure/azure-sdk-for-python/blob/main/doc/deprecation_process.md)

It's also the README title in generated packages. azure-mgmt-eventhub here for example: https://github.com/Azure/azure-sdk-for-python/blob/ba02dff56c1d52ba6ad369d8d01af3be733b4f68/sdk/eventhub/azure-mgmt-eventhub/README.md